src/components/WPScan/WPScan.tsx

```diff
@@ -160,136 +226,164 @@ const WPScan = () => {
-                <TextInput
-                    label={"URL of target wordpress site"}
-                    placeholder={"Example: http://www.wordpress.com/sample"}
-                    required
-                    {...form.getInputProps("url")}
-                />
-                <Switch
-                    size="md"
-                    label="Advanced Mode"
-                    checked={checkedAdvanced}
-                    onChange={(e) => setCheckedAdvanced(e.currentTarget.checked)}
-                />
-                {checkedAdvanced && (
-                    <>
-                        <Switch
-                            size="md"
-                            label="Stealthy"
-                            checked={stealthyChecked}
-                            onChange={(e) => setStealthyChecked(e.currentTarget.checked)}
-                        />
-                        <Switch
-                            size="md"
-                            label="Verbose"
-                            checked={verboseChecked}
-                            onChange={(e) => setVerboseChecked(e.currentTarget.checked)}
-                        />
-                        <NativeSelect
-                            value={selectedEnumerationType}
-                            onChange={(e) => setselectedEnumerationType(e.target.value)}
-                            title={"Enumeration Options"}
-                            data={enumerationtypes}
-                            placeholder={"Types"}
-                            description={"Please select an enumeration type"}
-                        />
-                        {enumerationRequiringRange.includes(selectedEnumerationType) && (
-                            <>
-                                <Grid>
-                                    <Grid.Col span={6}>
-                                        <NumberInput
-                                            label={"Lower Range"}
-                                            placeholder={"e.g. 1"}
-                                            {...form.getInputProps("lowbound")}
-                                        />
-                                    </Grid.Col>
+        <RenderComponent
+            title={title}
+            description={description}
+            steps={steps}
+            tutorial={tutorial}
+            sourceLink={sourceLink}
+        >
+            {!loadingModal && (
+                <InstallationModal
+                    isOpen={opened}
+                    setOpened={setOpened}
+                    feature_description={description}
+                    dependencies={dependencies}
+                ></InstallationModal>
+            )}
+            <form onSubmit={form.onSubmit((values) => onSubmit(values))}>
+                {LoadingOverlayAndCancelButton(loading, pid)}
+                <Stack>
+                    <TextInput
+                        label={"URL of target wordpress site"}
```

Comment:
Make it WordPress so it's consistent with the steps and description. 
```suggestion
                        label={"URL of target WordPress site"}
```